apps/test/unit/storage/dataBrowser/dataVisualizer/VisualizerModalTest.js

```diff
@@ -77,6 +81,88 @@ describe('VisualizerModal', () => {
     });
   });
 
+  describe('filtering', () => {
+    let wrapper;
+    beforeEach(() => {
+      wrapper = mount(<VisualizerModal {...DEFAULT_PROPS} />);
+      wrapper.instance().handleOpen();
+    });
+
+    it('finds all possible values in the filter column', () => {
+      let records = [
+        {id: 1, name: 'alice', age: 7, male: false},
+        {id: 2, name: 'bob', age: 8, male: true},
+        {id: 3, name: 'charlie', age: 9, male: true}
+      ];
+      expect(
+        wrapper.instance().getValuesForFilterColumn(records, 'name')
+      ).to.deep.equal(['alice', 'bob', 'charlie']);
+      expect(
+        wrapper.instance().getValuesForFilterColumn(records, 'age')
+      ).to.deep.equal([7, 8, 9]);
+      expect(
+        wrapper.instance().getValuesForFilterColumn(records, 'male')
+      ).to.deep.equal([false, true]);
+    });
+
+    it('sorts the filter values alphabetically or numerically', () => {
+      let records = [
+        {id: 2, name: 'bob', age: 8, male: true},
+        {id: 3, name: 'charlie', age: 9, male: true},
+        {id: 1, name: 'alice', age: 7, male: false}
+      ];
+      expect(
+        wrapper.instance().getValuesForFilterColumn(records, 'name')
+      ).to.deep.equal(['alice', 'bob', 'charlie']);
+      expect(
+        wrapper.instance().getValuesForFilterColumn(records, 'age')
+      ).to.deep.equal([7, 8, 9]);
```

Comment:
What's the expected behavior for the `male` column here? (My guess: `[false, true]`)